### ISSUE TITLE
Undefined function 'VineCopulaMatlab'

### ISSUE THREAD

**Issue by milsppp** [MalteKurz/VineCopulaMatlab#11]
I get the following error when I try to run 'demoVineCopulaMatlab.m'. I would be grateful for help with solving this problem as this toolbox looks very useful. Many thank 

Undefined function 'VineCopulaMatlab' for input arguments of type 'double'.

Error in VineCopulaMatlabSetSeed (line 7)
    SeedState = VineCopulaMatlab(1005);

Error in VineCopulaMatlabSeedState (line 22)
    SeedState = VineCopulaMatlabSetSeed('shuffle');

Error in VineCopulaRand (line 159)
    SeedState = VineCopulaMatlabSeedState;

Error in VineCopula/Sim (line 1264)
                    U =
                    VineCopulaRand(obj.type,N,obj.dimension,obj.families(1,:),parameters,obj.simplified,[],obj.families(2,:));
                    
Error in demoVineCopulaMatlab (line 57)
U = Sim(VineCopulaObject1,500);

**Comment by MalteKurz** [comment]
* looks like you haven't compiled the c++ dependencies yet.
* check out the docu here: http://maltekurz.github.io/VineCopulaMatlab/installVineCPP/

**Comment by oevkaya** [comment]
Dear Malte,
I got the same error when I try to make a simulation but it seems I have a problem on 3. issue on the list of install. under windows in the suggested website http://maltekurz.github.io/VineCopulaMatlab/installVineCPP/
I set up MingGW-w64, but in the command prompt I could not figure out how to finish the rest for installation


![2020-07-10_20-09-51](https://user-images.githubusercontent.com/68122167/87180599-83a4b700-c2e9-11ea-8848-43af78050ac8.png)

**Comment by MalteKurz** [comment]
I assume with step 3. you mean the installation of NLopt. If so, you may want to take a look here https://nlopt.readthedocs.io/en/latest/NLopt_on_Windows/#nlopt-with-mingw

**Comment by oevkaya** [comment]
> I assume with step 3. you mean the installation of NLopt. If so, you may want to take a look here https://nlopt.readthedocs.io/en/latest/NLopt_on_Windows/#nlopt-with-mingw

Yes, I am trying to follow your steps but, certainly as a really new learner of C++, I have real troubles about the installation. 
Hoping that I can see the last step ASAP and simulate non-simplified data with the toolbox. 
Thanks for your response Malte, kind regards...
